client/reducers/initialState.js

```diff
@@ -1,9 +1,14 @@
 export default {
-  access: { isAuthenticated: false, user: {} },
+  access: {
```

Comment:
What's the difference between the user object in `access` and the `activeUser` object?